python/ray/tests/test_autoscaler_e2e.py

```diff
@@ -91,7 +91,11 @@ def ping(self):
                 return True
 
         zero_reported_condition = get_metric_check_condition(
-            {"autoscaler_cluster_resources": 0, "autoscaler_pending_resources": 0},
+            {
+                "autoscaler_cluster_resources": 0,
+                "autoscaler_pending_resources": 0,
+                "autoscaler_pending_nodes": 0,
```

Comment:
Any way to write a test that could've caught this?

Reply:
isn't this precisely a test that catches this?